src/utils/goodsPrepared.js

```diff
@@ -0,0 +1,30 @@
+import { SORT_BY_ALPHABHET, SORT_BY_LENGTH, SORT_BY_NONE } from './vars';
+
+export const getPreparedGoods = (goods, { sortField, isReversed }) => {
+  if (sortField === SORT_BY_NONE && isReversed === '') {
+    return goods;
+  }
+
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((goods1, goods2) => {
```

Comment:
let's use 'a' and 'b' instead of good1 and good2, using numbers in variable names is not the best practice, everyone knows that 'a' and 'b' are goods